src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -150,6 +133,27 @@ public class ClusterTransactionCompletionResult
             public DynamicJsonArray Array;
         }
 
+
+        private void BatchTrafficWatch(ArraySegment<BatchRequestParser.CommandData> parsedCommands)
+        {
+            var sb = new StringBuilder();
+            for (var i = parsedCommands.Offset; i < (parsedCommands.Offset + parsedCommands.Count); i++)
+            {
+                // watching only patch commands
+                if (parsedCommands.Array[i].Patch != null)
```

Comment:
Why are you watching only patches?

Reply:
I could log the documents that are added via this function (Task BulkDocs()), I am already iterating through this array.

Reply:
Yes, the idea is that you would write `PUT users/1`, `DEL users/2`, etc for each of the commands in this case

Reply:
For patch commands would you like to see a simple "PATCH users/1" 
or "PATCH users/1 Script: ... Values: ..."
?